app/rpc/rpchandlers/get_peer_addresses.go

```diff
@@ -11,9 +11,11 @@ import (
 
 // HandleGetPeerAddresses handles the respectively named RPC command
 func HandleGetPeerAddresses(context *rpccontext.Context, _ *router.Router, _ appmessage.Message) (appmessage.Message, error) {
-	netAaddresses := context.AddressManager.Addresses()
-	addresses := make([]*appmessage.GetPeerAddressesKnownAddressMessage, len(netAaddresses))
-	for i, netAddress := range netAaddresses {
+	netAddresses := context.AddressManager.Addresses()
+	netAddresses = append(netAddresses, context.AddressManager.BannedAddresses()...)
```

Comment:
As previously mentioned, the RPC needs to return the banned addresses in a separate list.
Please add new field in the RPC response for "Banned Addresses", and place the output of `context.AddressManager.BannedAddresses` in it.